HCL.ArticleService.API.Domain/DTO/ArticleDTO.cs

```diff
@@ -8,9 +9,16 @@ namespace HCL.ArticleService.API.Domain.DTO
 {
     public class ArticleDTO
     {
+        [Required(AllowEmptyStrings = true, ErrorMessage = "Need Title")]
```

Comment:
Why do you decide to use attributes?

Reply:
because i want the model to be required.
I forget add model validation to api